tests/test_format_alerts.py

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/env python3
+# -*- coding: UTF-8 -*-
+
+"""This file provides tests for the alert formatting performed prior to storage
+in GCS. This includes testing the generation of the valid schema pickle file by
+the ``alert_ingestion.valid_schemas.gen_valid_schema`` module, and the
+correction of alert schemas by the ``alert_ingestion.consume`` module.
+
+Before running these tests:
```

Comment:
This section might be useful to move into `tests/__init__.py` but I leave it at your discretion.